superset-frontend/src/dashboard/components/nativeFilters/FilterBar/FilterSets/FilterSets.tsx

```diff
@@ -125,15 +129,14 @@ const FilterSets: React.FC<FilterSetsProps> = ({
     if (!id) {
       return;
     }
+
     const filtersSet = filterSets[id];
+
     Object.values(filtersSet?.dataMask?.nativeFilters ?? []).forEach(
       dataMask => {
         const { extraFormData, currentState, id } = dataMask as MaskWithId;
         // if we have extra filters in filter set don't add them to selected data mask || if we have filters with changed metadata not apply them
-        if (
```

Comment:
I think we can remove the comment on the line above (the helper seems fairly self-explanatory) - alternatively rename the helper to something so obvious the comment won't be needed (`isFilterMissingOrContainsInvalidMetadata`)